kodilla-exception/src/main/java/com/kodilla/exception/test/FlightSchedule.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.exception.test;
+
+import java.util.*;
+
+public class FlightSchedule {
+    public boolean findFilght(Flight flight) throws RouteNotFoundException {
+        if (flight == null) {
+            throw new IllegalArgumentException();
+        }
+
+        Map<String, Boolean> arrivalAbilities = new HashMap<>();
```

Comment:
populowanie mapy mogłoby być w konstruktorze - dzięki temu metoda findFlight byłaby znaczniej krótsza i bardziej wydajna - obecnie za każdym razem wywołania tej metody tworzymy tę mapę i dodajemy do niej wartości - a wystarczyłoby to zrobić tylko raz.